robust_pdf_plot.ipynb

```diff
@@ -0,0 +1,311 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #122.        ax.set_ylabel("$PDF$")
</pre><p>Ideally we should use "pdf" without math symbol ($)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Done</p>